src/constants.rs

```diff
@@ -35,6 +35,9 @@ pub const A: FieldElement       = FieldElement([
     486662, 0, 0, 0, 0, 0, 0, 0, 0, 0, ]);
 
 /// SQRT_MINUS_A is sqrt(-486662)
+// XXX I think that this was used in Adam's code for his elligator
+// implementation, but that should maybe be using sqrt(-486664)
+// instead...?  - hdevalence
```

Comment:
@hdevalence Maybe we should try asking Adam how he derived this? And maybe in a more polite manner this time than ambushing him on the metro.